modules/plugins/theme/theme.nix

```diff
@@ -3,33 +3,48 @@
   lib,
   ...
 }: let
+  inherit (builtins) listToAttrs;
   inherit (lib.options) mkOption;
   inherit (lib.attrsets) attrNames;
```

Comment:
`lib.attrsets` is already in scope, move `listToAttrs` into `inherit (lib.attrsets)`